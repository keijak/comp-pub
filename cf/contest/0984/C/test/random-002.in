20
25 188 62
93 99 199
191 156 164
151 197 7
176 70 25
50 51 72
57 20 89
75 151 37
49 99 148
20 187 144
13 106 158
69 43 164
45 177 135
56 168 44
167 120 190
17 124 110
173 95 49
112 61 77
14 130 21
171 47 136
