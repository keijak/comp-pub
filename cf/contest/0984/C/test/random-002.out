Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
